src/verifact_agents/evidence_hunter.py

```
@@ -10,46 +15,194 @@
+        list[str]: A list of trust sources.
+    """
+    with open(path, "r", encoding="utf-8") as f:
+        return [
+            line.strip()
+            for line in f
+            if line.strip() and not line.strip().startswith("#")
+        ]
+
+class EvidenceHunter:
+
+    def __init__(self,trust_sources_path: str="data\\trust_sources.txt"):
```

Ruff (0.11.9)

Missing argument description in the docstring for `__init__`: `trust_sources_path`

(D417)